modules/openapi-generator/src/main/resources/cpp-qt5-client/helpers-body.mustache

```diff
@@ -7,14 +7,29 @@
 namespace {{this}} {
 {{/cppNamespaceDeclarations}}
 
+struct {{prefix}}CustomDateTimeFormat{
+    bool setString = false;
+    QString formatString;
+    bool setEnum = false;
+    Qt::DateFormat formatEnum;
+};
+
 class {{prefix}}SerializerSettings {
 public:
-    static void setDateTimeFormat(const QString & dtFormat){
-        getInstance()->dateTimeFormat = dtFormat;
+    static {{prefix}}CustomDateTimeFormat getCustomDateTimeFormat() {
+        return getInstance()->customDateTimeFormat;
     }
-    static QString getDateTimeFormat() {
-        return getInstance()->dateTimeFormat;
+
+    static void setDateTimeFormatString(const QString &dtFormat){
+        getInstance()->customDateTimeFormat.setString = true;
```

Comment:
Here maybe you can set the other flag as false, otherwise the check in the other API will have no effect.

Reply:
done